### PR TITLE
docs: Updated top level docs readme

### PR DESCRIPTION
**Pull request by neeraj-laad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Appsody Documentation
+
+Welcome to the source repository for [Appsody documentation](https://appsody.dev/docs/)
```
Comment (helenmasters) [inline README.md]: Welcome to the GitHub repository for [Appsody documentation](https://appsody.dev/docs/).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Documentation source files
+
+All source files for the documentation on Appsody website are located within the `/docs` folder of this repository. These source files are orgnized into a folders that represent a category within the published documentation.
```
Comment (helenmasters) [inline README.md]: All the source files for the documentation on the [Appsody website](https://appsody.dev) are located within the `/docs` folder of this repository. The source files are organized into subfolders that represent a category within the published documentation.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Improving our documentation
```
Comment (helenmasters) [inline README.md]: Why don't we call this 
## Contributing?
Then you could have.... 
"We welcome all contributions to the Appsody project. Please see our Contributing Guidelines."
(where Contributing Guidelines links to the CONTRIBUTING.md).

If you find a bug, such as a typo, or are confused by our documentation, please let us know by [creating an issue](https://github.com/appsody/docs/issues/new).

## Community

Join our [Slack] to meet the team, ask for help, or talk about Appsody!

Comment (neeraj-laad) [reply README.md]: I like this. updated.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## Documentation source files
+
+All the source files for the documentation on the [Appsody website](https://appsody.dev) are located within the `/docs` folder of this repository. The source files are organized into subfolders that represent a category within the published documentation.
+
+## Contributing
+
+We welcome all contributions to the Appsody project. Please see our [Contributing guidelines](CONTRIBUTING.md)
+
+IIf you find a bug, such as a typo, or are confused by our documentation, please let us know by [creating an issue](https://github.com/appsody/docs/issues/new).
```
Comment (helenmasters) [inline README.md]: IIf?

**Pull request merged**